source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_helper.py

```diff
@@ -66,13 +68,143 @@ def build_ridesharing_crowfly_journey(instance, origin, destination, period_extr
         ridesharing_section.end_date_time = period_extremity.datetime
 
     # report section values into journey
-    ridesharing_journey.tags.append('ridesharing')
     ridesharing_journey.distances.ridesharing = ridesharing_section.length
     ridesharing_journey.durations.ridesharing = ridesharing_section.duration
     ridesharing_journey.duration = ridesharing_section.duration
     ridesharing_journey.durations.total = ridesharing_section.duration
-    ridesharing_journey.requested_date_time = period_extremity.datetime
     ridesharing_journey.departure_date_time = ridesharing_section.begin_date_time
     ridesharing_journey.arrival_date_time = ridesharing_section.end_date_time
 
-    return ridesharing_journey
+    pb_rsjs, pb_tickets = build_ridesharing_journeys(from_pt_obj=ridesharing_section.origin,
+                                         to_pt_obj=ridesharing_section.destination,
+                                         period_extremity=period_extremity,
+                                         instance=instance)
+    if not pb_rsjs:
+        return None
+
+    ridesharing_section.ridesharing_journeys.extend(pb_rsjs)
+    return ridesharing_journey, pb_tickets
+
+
+def build_ridesharing_journeys(from_pt_obj, to_pt_obj, period_extremity, instance):
+    logger = logging.getLogger(__name__)
+
+    from_coord = get_pt_object_coord(from_pt_obj)
+    to_coord = get_pt_object_coord(to_pt_obj)
+    from_str="{},{}".format(from_coord.lat, from_coord.lon)
```

Comment:
we should create a class Coord and implment  `__repr__` to avoid doing '{},{}' everywhere manually, especially, we tend to inverse lat and lon, obviously, it can be done later